### PR TITLE
Jitter: label to_string

### PR DESCRIPTION
**Pull request by serpilliere**

### CODE REVIEW

File: miasm2/core/asmblock.py [hunk 1]
```diff
@@ -67,8 +67,17 @@ def set_label(self, loc_key):
 
     label = property(get_label, set_label)
 
+    def to_string(self, symbol_pool=None):
+        if symbol_pool is None:
+            return "%s:%s" % (str(self.c_t), str(self.loc_key))
```
Comment (commial) [inline miasm2/core/asmblock.py]: `str`s are useless, due to string format

File: miasm2/core/asmblock.py [hunk 2]
```diff
@@ -67,8 +67,17 @@ def set_label(self, loc_key):
...
+        else:
+            return "%s:%s" % (
+                str(self.c_t),
```
Comment (commial) [inline miasm2/core/asmblock.py]: Idem (and `c_t` is already a string)

File: miasm2/core/asmblock.py (hunk 1 above)
Comment (serpilliere) [reply miasm2/core/asmblock.py]: done

File: miasm2/core/asmblock.py (hunk 2 above)
Comment (serpilliere) [reply miasm2/core/asmblock.py]: done

**Pull request merged**